qhana_plugin_runner/util/request_helpers.py

```diff
@@ -63,7 +64,11 @@ def send(self, request: PreparedRequest, stream: bool, **kwargs) -> Response:
                 resp.headers["Content-Length"] = str(file_path.stat().st_size)
                 resp.status_code = HTTPStatus.OK
 
-                # TODO set mimetype in response?
+                # set mimetype in response if guessable
+                mimetype = mimetypes.MimeTypes().guess_type(url=file_path)[0]
+                if mimetype:
+                    resp.headers["Content-Type"] = mimetype
```

Comment:
```suggestion
                mimetypes = mimetypes.MimeTypes().guess_type(url=file_path)
                if mimetypes:
                    resp.headers["Content-Type"] = mimetypes[0]
```
Should be safer as the index access happens after the not none/empty check.